tests/baselines/reference/ES5SymbolProperty5.types

```diff
@@ -14,8 +14,8 @@ class C {
 }
 
 (new C)[Symbol.iterator](0) // Should error

```

Comment:
`Symbol.iterator` in this case comes from an anonymous type on a `Symbol` variable declaration, so the interop `symbol` -> `unique symbol` code path doesn;t take effect. So the `Symbol.iterator` in the class body makes an index signature, and that index signature doesn't handle symbols because they never do, hence the failed lookup here. It's not a problem that arises in the real world, since in the real world, the `lib` defines the global `Symbol` type using the `SymbolConstructor` type which we apply interop logic to.

Reply:
So....what is this test testing? Anything useful? Maybe it could be removed?

Reply:
If/when we get proper symbol index signature support, its behavior should change, so I don't think it should be removed.